src/controllers/EventsController.php

```diff
@@ -30,14 +30,12 @@ public function getAction($request, $db)
                     );
                     break;
                 case 'talk_comments':
-                    $sort                = $this->getSort($request);
                     $talk_comment_mapper = new TalkCommentMapper($db, $request);
                     $list                = $talk_comment_mapper->getCommentsByEventId(
```

Comment:
The method [`TalkCommentMapper::getCommentsByEventId()`](https://github.com/joindin/joindin-api/blob/dd87d0258a8e0a0dec41da585f6574034295ccef/src/models/TalkCommentMapper.php#L56-L79) accepts only 4 parameters; `$sort` is never used.